### DISCUSSION TITLE
Add sdist to pypi

### DISCUSSION THREAD

**Discussion by julian-belina** [ERGO-Code/HiGHS#1426] [Announcements]
Is it possible to upload the source distribution addtionally to the wheel to pypi? The source distribution is required to create a conda package from the pypi package. This would really simpify one of the pipelines that i maintain.

**Comment by jajhall** [comment]
We're aware of this, thanks, and it's on the to do list of @galabovaa

**Reply by Midnighter** [reply]
I'm also interested in creating a conda-forge package for highspy, but the sources are not yet available. Is there something that we can do to help?

**Reply by Midnighter** [reply]
@galabovaa I find the title of this discussion a bit misleading, as a source distribution is important for any platform.

Anyway, I wanted to check back and see where you are with this issue? Having a conda package for highspy is an important building block for getting another tool of mine on bioconda. Thank you for your consideration.

**Reply by julian-belina** [reply]
You are right. The source distribution is not specific to the windows installation. I hope the new title is more descriptive

**Reply by sscherfke** [reply]
Could this please be done? If you can create wheels, creating and uploading an sdist should not really be a problem.

**Reply by jajhall** [reply]
Sorry this has slipped by. I'll remind @galabovaa, and if it's as simple as you say, I'm sure it will be done without further undue delay

**Comment by galabovaa** [comment]
The source distribution has been added to our latest release. Please let us know if you run into any further issues!
